pi-manage

```diff
@@ -275,11 +279,13 @@ def create(directory="/var/lib/privacyidea/backup/",
         sqlfile = "%s/db-%s.sqlite" % (directory, DATE)
         call(["cp", productive_file, sqlfile])
     elif sqltype in MYSQL_DIALECTS:
-        m = re.match(".*mysql://(.*):(.*)@(.*)/(.*)", sqluri)
+        m = re.match(".*mysql://(.*):(.*)@(.*)/(\w*)\??(.*)", sqluri)
         username = m.groups()[0]
         password = m.groups()[1]
         datahost = m.groups()[2]
         database = m.groups()[3]
+        # We strip parameters, but we do not use them
+        _parameters = m.groups()[4]
         defaults_file = "{0!s}/mysql.cnf".format(conf_dir)
         _write_mysql_defaults(defaults_file, username, password)
         call("mysqldump --defaults-file=%s -h %s %s > %s" % (
```

Comment:
Change looks good to me, but I think it would be nice to get rid of the unescaped variables in the subprocess calls with ``shell=True`` in this function. We could do this using ``pipes.quote``.
In order to stay compatible to Python 2+3, we could use ``from six.moves import shlex_quote``.

But I can also do this in a separate PR.

Reply:
As I do not exactly understand what you mean, an addtional PR would be great! ;-)

Reply:
Will do!